docs/terminology.md

```diff
@@ -38,3 +38,11 @@ A **task** specifies a desired annotation result. A **task instance** refers to
 A **project** is a collection of tasks.
 
 See also the [task and projects guide](./tasks.md).
+
+## Segments
+At its lowest-level a **segment** is the collection of several annotated voxels. At a larger level, segments can grow to be the size of a whole cell bodies or partial cells, e.g. a single axon.
+
+Typically many segments make up a segmentation. Segments can be painted manually using the webKnossos volume annotation tools or created through third-party programs typically resulting in larger segmentations of a dataset.
```

Comment:
Well, yes and no. I want to explicitly make the connection between `segments` and `segmentations`, especially since we have `Segmentation` layers as first-order primitives in wK.